packages/sdk-v2/package.json

```diff
@@ -42,6 +42,9 @@
     "@types/node": "18.0.0",
     "@types/react-native": "0.70.4",
     "@usedapp/core": "^1.2.13",
+    "@veramo/core-types": "5.6.1-next.45",
```

Comment:
remove all new dependencies.
we just store jsons
I dont want any dependency on veramo

Reply:
yes, is left-over dependency from when I was testing a local resolver. should be removed

Reply:
When you create a PR, you check everything before you commit
you do not commit things you will "later" remove
after you create a PR you go over the changed files and make sure there are no such "later"